tests/integration/standard/test_policies.py

```diff
@@ -161,3 +165,42 @@ def test_end_to_end_simple(self):
         (encrypted,unencrypted) = session.execute(prepared, [expected]).one()
         self.assertEquals(expected, encrypted)
         self.assertEquals(expected, unencrypted)
+
+    def test_end_to_end_different_cle_contexts(self):
```

Comment:
When run manually this test fails (with the expected errors) without the rest of the changes in this PR and passes with them.